### PR TITLE
add migration workload

### PR DESCRIPTION
**Pull request by bbeaudreault**
This was surprisingly easy. I just want to figure out how to add an endtoend test.

### CODE REVIEW

File: proto/query.proto [hunk 1]
```diff
@@ -262,6 +262,7 @@ message ExecuteOptions {
     UNSPECIFIED = 0;
     OLTP = 1;
     OLAP = 2;
+    MIGRATION = 3;
```
Comment (sougou) [inline proto/query.proto]: Do you think `DBA` will be better? It's basically for maintenance tasks, right?

Comment (bbeaudreault) [inline proto/query.proto]: Will do. That name fits better in the vitess nomenclature.  I'll also fix the test.

**Comment by bbeaudreault** [comment]
@sougou can we merge this? I'd like to build the java side but need the protos to be in master so i can rebase nicely.

**Comment by bbeaudreault** [comment]
oh apparently there's a conflict.  one min.

**Comment by sougou** [comment]
I think you accidentally canceled out the DBA changes? It's back to 'migration' and some tests are failing.

**Comment by bbeaudreault** [comment]
Ugh sorry. Have been juggling a lot today. Will fix tomorrow

**Comment by bbeaudreault** [comment]
@sougou I fixed the test. this should be good to merge now

**Comment by sougou** [comment]
LGTM

**Pull request merged**